components/story-element.js

```diff
@@ -71,6 +71,12 @@ function StoryElementTable({element}) {
   return React.createElement(Table, { id: element.id, data: element.data, hasHeader: element.metadata['has-header']});
 }
 
+function StoryElementFile({element}) {
+  return React.createElement("div", {className: "attachment", key: element.id}, 
+            React.createElement("div", {className: "attachment__title"}, element["file-name"]),
+            React.createElement("a", {className: "attachment__link", href: element.url, download: true}, 'download'))
+}
```

Comment:
I think the classnames we followed have a `story-element-` added as a prefix. Like `story-element-attachment`. Refer: https://github.com/quintype/quintype-node-components/blob/c55b46909c6ea95a56b687e253a48331062a239e/components/story-element.js#L109 @munirajun Please correct me if I am wrong